site/content/docs/main/performance-guidance.md

```diff
@@ -0,0 +1,165 @@
+#### result:
+|Uploader| Resources|Times |Max CPU|Max Memory|Repo Usage|
+|--------|----------|:----:|------:|:--------:|:--------:|
+| Kopia  | 1c1g     |5m06s | 71%   |861 MB    |108 MB    |
+| Restic | 1c1g     |6m23s | 56%   |810 MB    |275 MB    |
+
+### Case 2.4 2097152(2M) files, 2396745(2M) directories, 100B per file total 200.000MB content
+#### result:
+|Uploader| Resources|Times |Max CPU|Max Memory|Repo Usage|
+|--------|----------|:----:|------:|:--------:|:--------:|
+| Kopia  | 1c1g     |OOM   | 74%   |N/A       |N/A       |
+| Restic | 1c1g     |41m47s| 52%   |904 MB    |3.2 GB    |
+#### conclusion:
+- With the increasing number of files, there is no memory abnormal surge, the memory usage for both Kopia uploader and Restic uploader is linear increasing, until exceeds 1GB memory usage in Case 2.4 Kopia uploader OOM happened.
+- Kopia uploader gets increasingly faster along with the increasing number of files.
+- Restic uploader repository size is still much larger than Kopia uploader repository.
+
+### Case 3: 10625(10k) files, 781 directories, 1.000MB per file total 10.376GB content
+#### result:
+|Uploader| Resources|Times |Max CPU|Max Memory|Repo Usage|
+|--------|----------|:----:|------:|:--------:|:--------:|
+| Kopia  | 1c2g     |1m37s | 75%   |251 MB    |10 GB     |
+| Restic | 1c2g     |5m25s | 100%  |153 MB    |10 GB     |
+| Kopia  | 4c4g     |1m35s | 75%   |248 MB    |10 GB     |
+| Restic | 4c4g     |3m17s | 171%  |126 MB    |10 GB     |
+#### conclusion:
+- This case involves a relatively large backup size, there is no significant time reduction by increasing resources from 1c2g to 4c4g for Kopia uploader, but for Restic upoader when increasing CPU from 1 core to 4, backup time-consuming was shortened by one-third, which means in this scenario should allocate more CPU resources for Restic uploader.
+- For the large backup size case, Restic uploader's repository size comes to normal
+
+### Case 4: 900 files, 1 directory, 1.000GB per file total 900.000GB content
+#### result:
+|Uploader| Resources|Times  |Max CPU|Max Memory|Repo Usage|
+|--------|----------|:-----:|------:|:--------:|:--------:|
+| Kopia  | 1c2g     |2h30m  | 100%  |714 MB   |900 GB     |
+| Restic | 1c2g     |Timeout| 100%  |416 MB   |N/A        |
+| Kopia  | 4c4g     |1h42m  | 138%  |786 MB   |900 GB     |
+| Restic | 4c4g     |2h15m  | 351%  |606 MB   |900 GB     |
+#### conclusion:
+- When the target backup data is relatively large, Restic uploader starts to Timeout under 1c2g. So it's better to allocate more memory for Restic uploader when backup large sizes of data.
+- For backup large amounts of data, Kopia uploader is both less time-consuming and less resource usage.
+
+## Summary
+- With the same specification resources, Kopia uploader is less time-consuming when backup.
+- Choosing Kopia uploader is both less time-consuming and less resource usage if backup large amounts of data.
```

Comment:
How about this:
`Choosing Kopia uploader for the cases of large mounts of data or massive small files, in order to make your backup faster and less resource consuming.`

Reply:
for massive small files, kopia needs more CPU resources, so I only mentioned in backup large amounts of data scenario